app/src/main/res/values-fr/strings.xml

```diff
@@ -11,8 +11,320 @@
+  <string name="troubleshoot_server_unstable_details">
+      Le serveur répond actuellement avec le code d\'erreur 500, indiquant que le serveur peut être dans un état d\'erreur.
+      \n
+      \n S\'il vous plaît contacter un administrateur.</string>
+  <string name="troubleshoot_server_not_responding">Serveur ne répond pas</string>
+  <string name="troubleshoot_server_not_responding_details">
+      Le serveur est actuellement ne répond pas aux demandes. Cela peut être parce que:\n
+      \n • Le serveur est momentanément indisponible.
+      \n • Le serveur est en cours de démarrage.
+      \n
+      \n S\'il vous plaît contacter un administrateur.</string>
+  <string name="patient_location_error_unknown">Une erreur inconnue est survenue lors de la mise à jour se trouve le patient. Veuillez réessayer.</string>
+  <string name="patient_location_error_interrupted">Patient mise à jour de l\'emplacement a été interrompu. Veuillez réessayer.</string>
+  <string name="patient_location_error_network">Erreur de mise à jour du patient; s\'il vous plaît vérifier que vous êtes connecté au serveur.</string>
+  <string name="patient_location_error_no_such_patient">Mise à jour de localisation d\'un patient Erreur: ce patient ne existe pas sur le serveur.</string>
+  <string name="troubleshoot_package_server_unreachable">Serveur de Package inaccessible</string>
+  <string name="troubleshoot_update_server_unreachable_details">
+      Le serveur de package n\'a pas pu être atteint. Cela peut être parce que:
+      \n
+      \n • L\'URL du serveur est incorrect.
+      \n • Le serveur est en panne.
+      \n
+      \n S\'il vous plaît corriger les paramètres du serveur de colis ou consulter un administrateur de l\'aide.
+  </string>
+  <string name="troubleshoot_package_server_misconfigured">Paramètre de serveur de package peut être incorrect</string>
+  <string name="troubleshoot_update_server_misconfigured_details">
+      S\'il vous plaît vérifier que les paramètres du serveur de package est correct ou consultez un administrateur de l\'aide.
+  </string>
+
+  <string name="present_filter_description">Tous les présents patients</string>
+  <string name="age_filter_description">Sous %d années</string>
+  <string name="filter_group_default_name">Plusieurs filtres</string>
+
+  <string name="patient_missing_age">Entrez l\'âge (en utilisant années, mois, ou les deux)</string>
+  <string name="patient_validation_missing_family_name">Entrez le nom de famille</string>
+  <string name="patient_validation_missing_given_name">Entrez le nom donné</string>
+  <string name="patient_validation_missing_id">Entrez l\'ID du patient</string>
+  <string name="patient_validation_missing_admission_date">Entrez la date d\'admission</string>
+  <string name="patient_missing_gender">S\'il vous plaît sélectionnez Masculin ou Féminin.</string>
+  <string name="patient_validation_future_admission_date">Date d\'admission ne peut pas être dans l\'avenir.</string>
```

Comment:
i`m not sure .. maybe the proper preposition for 'être dans l\'avenir'  would be  "être à l\'avenir"